getting-started/vs-code-integration/scaffolders.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Telerik UI for Blazor Code Scaffolders
+page_title: Telerik UI for Blazor Code Scaffolders - Visual Studio Code Integration
+description: Learn how to add a new component page in Visual Studio Code via code scaffolder templates.
+slug: getting-started-vs-code-integration-scaffolders
+position: 20
+---
+
+
+# Scaffolders
+
+The Telerik Visual Studio Code extension provides a handy feature for increased developer productivity – scaffolders. They allow you to quickly add component pages to your application and setup the parameters of the component through an UI.
+
+## Get the Extension
+
+To use the **Telerik UI for Blazor Scaffolders**, install the Telerik UI for Blazor Visual Studio Code Extension. @[template](/_contentTemplates/common/general-info.md#vs-code-x-download)
+
+## Add a new component page
+
+1. Right-click on the name of the solution
+1. From the popup menu select the ``
+1. From the UI form, select the desired component and set the available parameters
+
+>caption Add a new component page
+
+## Available Scaffolded Components
+
+| Component |
```

Comment:
Why not sort the components alphabetically?

Reply:
Did it by component "popularity", started with the grid and everything else is listed below.